internal_transfer_with_agreed_amount/i18n/es.po

```diff
@@ -0,0 +1,129 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* internal_transfer_with_agreed_amount
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 16.0+e\n"
```

Comment:
Missing updating translations. At the OCA this step is not required because they handle translations on a separate place (Weblate), but we don't.